src/applications/proxy-rewrite/proxy-rewrite-entry.jsx

```diff
@@ -231,18 +231,12 @@ const scriptPaths = [
 
 const linkPaths = ['/generated/styleConsolidated.css'];
 
-function removeInjectedHeaderFooter(
-  links = linkPaths,
-  scripts = scriptPaths,
-  docHead = document.head,
-) {
-  Array.from(docHead.getElementsByTagName('script'))
-    .filter(node => scripts.includes(node.getAttribute('src')))
-    .forEach(node => docHead.removeChild(node));
-
-  Array.from(docHead.getElementsByTagName('link'))
-    .filter(node => links.includes(node.getAttribute('href')))
-    .forEach(node => docHead.removeChild(node));
+function removeInjectedHeaderFooter(docHead = document.head) {
+  Array.from(
+    document.querySelectorAll(
+      'script[src*="va-gov-assets"],link[href*="va-gov-assets"]',
```

Comment:
This approach is simpler- all of our asset buckets contain `va-gov-assets` so it's easy to target scripts and links with an attribute selector.

Reply:
You can verify this works by navigating to www.va.gov/health and pasting this function in the console: 

```
const docHead = document.head;
Array.from(
    document.querySelectorAll(
      'script[src*="va-gov-assets"],link[href*="va-gov-assets"]',
    ),
  ).forEach(node => docHead.removeChild(node));
```